portfolio/src/main/webapp/contact.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="contact.css">
+    <script src="script.js"></script>
+  </head>
+  <body>
+    <div class="contact_content">
+    </div>
```

Comment:
This is an empty div, so just delete it.

Reply:
Removed empty div